package.json

```
@@ -11,7 +11,7 @@
     "fix:prettier": "prettier --write ./src",
     "start": "storybook dev -p 6006",
     "build": "npm run build:dist",
-    "build:dist": "rm -Rf dist && tsc",
+    "build:dist": "rm -Rf dist && tsc && cp -R src/theme/fonts dist/theme/fonts",
```

The asset: `man-default.png` failed when the `UserMenu` component tried to use it.

```suggestion
    "build:dist": "rm -Rf dist && tsc && cp -R src/theme/fonts dist/theme/fonts && cp -R src/Assets dist/Assets",
```